### DISCUSSION TITLE
Language support

### DISCUSSION THREAD

**Discussion by vmishev** [nunocoracao/blowfish#1273] [General]
I am having the following issue. Whenever I add another language, in my case bulgarian, I tried with different languages as well, Iget the following error:
hugo v0.101.0+extended linux/amd64 BuildDate=unknown
Error: Error building site: render of "sitemap" failed: "/home/vmishev/Projects/Hugo/dopek4/themes/blowfish/layouts/_default/sitemap.xml:13:28": execute of template failed: template: _default/sitemap.xml:13:28: executing "_default/sitemap.xml" at <.Language.LanguageCode>: can't evaluate field LanguageCode in type *langs.Language
What should I do

**Comment by FableFatale** [comment]
hi guy, do you solve above issue?

**Reply by fauziralpiandi** [reply]
i solved that problem yesterday, the solution was to create a ```sitemap.xml``` manually
